libs/server/Resp/KeyAdminCommands.cs

```diff
@@ -148,6 +148,36 @@ private bool NetworkEXPIRE<TGarnetApi>(RespCommand command, ref TGarnetApi stora
                 }
             }
 
+            if (parseState.Count > 3)
+            {
+                if (!parseState.TryGetEnum(3, true, out ExpireOption additionExpireOption) || !additionExpireOption.IsValid(ref parseState.GetArgSliceByRef(3)))
```

Comment:
We're eliminating the use of TryGetEnum since it's highly inefficient, please check out this PR - https://github.com/microsoft/garnet/pull/693

Reply:
Fixed